t/24-worker-webui-connection.t

```diff
@@ -119,10 +119,6 @@ $client->on(
         }
         return undef;
     }
-    sub skip_job {
-        my ($self, $job_id, $reason) = @_;
-        $self->skipped_jobs->{$job_id} = $reason;
-    }
```

Comment:
You can also remove `has skipped_jobs            => sub { {}; };`. (I've just tested it.)

Reply:
thx, done